app/serializers/user_profile_serializer.rb

```diff
@@ -1,12 +1,20 @@
 class UserProfileSerializer
   include JSONAPI::Serializer
-  attributes :name, :email, :id, :bio
+  attributes :name, :email, :id, :bio, :profile_image_url
 
   attribute :donations do |user|
     user.donations ? user.donations.map { |donation| DonationSerializer.new(donation).serializable_hash } : []
   end
 
   attribute :orders do |user|
     user.orders ? user.orders.map { |order| OrderSerializer.new(order).serializable_hash } : []
+
```

Comment:
attribute :orders do is not closed

```suggestion
end
```